tests/model/test_model.py

```diff
@@ -1199,19 +1203,80 @@ def test_update_star_status(self, mocker, model, event_op,
             'all': False,
         }
         mocker.patch('zulipterminal.model.Model.update_rendered_view')
+        set_count = mocker.patch('zulipterminal.model.set_count')
 
         model.update_message_flag_status(event)
 
         changed_ids = set(indexed_ids) & set(event_message_ids)
         for changed_id in changed_ids:
             assert model.index['messages'][changed_id]['flags'] == flags_after
         (model.update_rendered_view.
-         has_calls([mocker.call(changed_id) for changed_id in changed_ids]))
+         assert_has_calls([mocker.call(changed_id)
+                          for changed_id in changed_ids]))
 
```

Comment:
This was not being asserted. It's not related to this commit, I can add a separate commit if think it would be more appropriate